### PR TITLE
Add convenience methods for adding children to stacks

### PR DESCRIPTION
**Pull request by kylebshr**
We've mentioned upstreaming these in Market PRs - just copy-pasted @watt's implementation :)

### CODE REVIEW

**Review by kyleve — APPROVED** [review]

**Comment by kyleve** [comment]
Don't forget the changelog!

**Comment by kylebshr** [comment]
Was typing it when I got the push for that message 😄

File: BlueprintUI/Sources/Layout/Stack.swift [hunk 1]
```diff
@@ -82,6 +82,24 @@ extension StackElement {
         ))
     }
 
+
+    /// Convenience method for adding a child with a grow and shrink priority of 0.0
+    ///
+    /// - parameters:
+    ///   - child: The child element to add to this stack
+    ///
+    mutating func addFixed(child: Element) {
```
Comment (bencochran) [inline BlueprintUI/Sources/Layout/Stack.swift:91]: These should be `public`

**Review by watt — APPROVED** [review]

File: BlueprintUI/Sources/Layout/Stack.swift (hunk 1 above)
Comment (kylebshr) [reply BlueprintUI/Sources/Layout/Stack.swift]: doi! nice catch

**Pull request merged**